### ISSUE TITLE
MIT license

### ISSUE THREAD

**Issue by lizfranco** [graingert/WOW#26]
Can someone clarify for me the license of this library. Is it MIT? 
Why is the site UpLabs charging for a license?

**Comment by polluks2** [comment]
Indeed, it's [MIT](https://github.com/graingert/WOW/blob/master/LICENSE).
